lib/runners/processor/cpplint.rb

```diff
@@ -53,7 +53,7 @@ def analyze(changes)
       _stdout, stderr, status = capture3 analyzer_bin, *analyzer_options
 
       if [0, 1].include? status.exitstatus
-        xml_output = REXML::Document.new(stderr).root
+        xml_output = read_xml(stderr)
```

Comment:
[suggestion] I think we cannot use `read_xml` here because `Processor#read_xml` method returns non-nil value.
But `xml_output` is expected as a nil-able value.

```suggestion
        xml_root = Nokogiri::XML(stderr).root
```

Reply:
fixed it on https://github.com/sider/runners/pull/2427/commits/f7bd1aaec4b91a9a69f1c77b06ab868e164d3583
Thanks